.tekton/serverless-bundle-135-pull-request.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     - name: build-args
       value:
         - VERSION=1.35.0
-        - MUSTGATHER=brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather:v4.13
+        - CLI_ARTIFACTS=registry.redhat.io/openshift4/ose-cli-artifacts:v4.13
```

Comment:
we need the updated must-gather Dockerfile first (#3002)
/hold

Reply:
Chicken-egg problem as described in https://github.com/openshift-knative/serverless-operator/pull/3002#issuecomment-2462495352